src/v2/Apps/Consign/Components/SubmissionStepper.tsx

```diff
@@ -0,0 +1,53 @@
+import { Media } from "v2/Utils/Responsive"
+import { Box, Step, Stepper } from "@artsy/palette"
+import React, { FC } from "react"
+
+interface SubmissionStepperProps {
+  currentStep: "Artwork Details" | "Upload Photos" | "Contact Information"
+}
+
+function typedArray<T extends string>(...elems: T[]): T[] {
+  return elems
+}
+
+export const submissionFlowSteps = typedArray(
+  "Artwork Details",
+  "Upload Photos",
+  "Contact Information"
+)
+
+export const SubmissionStepper: FC<SubmissionStepperProps> = ({
+  currentStep,
+}) => {
+  const stepIndex = submissionFlowSteps.indexOf(currentStep)
+  return (
+    <>
+      <Media between={["xs", "md"]}>
+        <Box>
+          <Stepper
+            initialTabIndex={stepIndex}
+            currentStepIndex={stepIndex}
+            disableNavigation
+            autoScroll
+          >
+            {submissionFlowSteps.map(step => (
+              <Step name={step} key={step} />
+            ))}
+          </Stepper>
+        </Box>
+      </Media>
+      <Media greaterThan="sm">
```

Comment:
@Serge0n - Its somewhat hard to read / understand seeing `between="xs, md"` and `greaterThan="sm". Does this mean: "at sm" and "at md"? 

Better yet, any chance we can refactor this to follow the more common pattern of:

```
<Media at=''xs'>
  ...
 </Media>
 <Media greaterThan='xs'>
    ...
 </Media>
 ```

Which simply means: "on phones render A; on desktops render B"